docs/vdpa/README.md

```diff
@@ -0,0 +1,68 @@
+# Using vDPA devices in Kubernetes
+## Introduction to vDPA
+vDPA (Virtio DataPath Acceleration) is a technology that enables the acceleration
+of virtIO devices while allowing the implementations of such devices
+(e.g: NIC vendors) to use their own control plane.
+
+The consumers of the virtIO devices (VMs or containers) interact with the devices
+using the standard virtIO datapath and virtio-compatible control paths (virtIO, vhost).
+While the data-plane is mapped directly to the accelerator device, the contol-plane
+gets translated the vDPA kernel framework.
+
+The vDPA kernel framework is composed of a vdpa bus (/sys/bus/vdpa), vdpa devices
+(/sys/bus/vdpa/devices) and vdpa drivers (/sys/bus/vdpa/drivers).
+Currently, two vdpa drivers are implemented:
+*  virtio_vdpa: Exposes the device as a virtio-net netdev
+*  vhost_vdpa: Exposes the device as a vhost-vdpa device. This device uses an extension
+of the vhost-net protocol to allow userspace applications access the rings directly
+
+For more information about the vDPA framework, read the article on
+[LWN.net](https://lwn.net/Articles/816063/) or the blog series written by one of the
+main authors ([part 1](https://www.redhat.com/en/blog/vdpa-kernel-framework-part-1-vdpa-bus-abstracting-hardware),
+[part 2](https://www.redhat.com/en/blog/vdpa-kernel-framework-part-2-vdpa-bus-drivers-kernel-subsystem-interactions),
+[part3](https://www.redhat.com/en/blog/vdpa-kernel-framework-part-3-usage-vms-and-containers))
+
+## vDPA Management
+Currently, the management of vDPA devices is performed using the sysfs interface exposed
+by the vDPA Framework. However, in order to decouple the management of vdpa devices from
+the SR-IOV Device Plugin functionality, this low-level management is done in an external
+library called [go-vdpa](https://github.com/k8snetworkplumbingwg/govdpa).
+
+In the context of the SR-IOV Device Plugin and the SR-IOV CNI, the current plan is to
+support only 1:1 mappings between SR-IOV VFs and vDPA devices despite the fact that
+the vDPA Framework might support 1:N mappings.
+
+## Tested NICs:
+* Mellanox ConnectX®-6 DX *
```

Comment:
What is the trailing *? Does that mean all variants or was there a second bullet that didn't get fully deleted?

Reply:
Fixed